src/db_impl.h

```diff
@@ -36,6 +52,11 @@ class TitanDBImpl : public TitanDB {
 
   Status Close() override;
 
+  using TitanDB::DefaultColumnFamily;
+  ColumnFamilyHandle* DefaultColumnFamily() const override {
```

Comment:
why add it specifially

Reply:
It had some code there to help with debugging, removed before proposing this PR. Removing this now.

Reply:
seems you didn't push the commit